packages/terra-application-navigation/src/common/_PopupMenu.jsx

```diff
@@ -2,7 +2,8 @@ import React, {
   useRef,
```

Comment:
nitpick - let's import useContext
```suggestion
  useRef,
  useContext,
```

Reply:
I don't think it's wrong to use `React.useContext`. In this case i think either could be used but in some instances we use `React.useContext` to allow us to spy on the useContext method from the react object and mock it when we couldn't do that if we import it.

For instance: https://github.com/cerner/terra-framework/blob/main/packages/terra-application-utility/tests/jest/UtilityMenuDivider.test.jsx#L18

Reply:
arg, I keep on forgetting about the testing use cases. I think this should be seared in my brain now lol